ui/src/app/shared/header/header.component.ts

```diff
@@ -174,8 +174,13 @@ export class HeaderComponent implements OnInit, AfterViewInit, AfterViewChecked,
   }
 
   removeChip(chipKey: string): void {
-    this.chips.delete(chipKey);
-    this.search();
+    if (this.chips.get(chipKey).valueOf()) {
+      this.chips.delete(chipKey);
+      this.search();
```

Comment:
Per the other ticket about "?q=", this special-casing shouldn't be necessary if we omit q= on an unfiltered query. I would just fix that other issue and leave this part of the code unchanged.